package.json

```diff
@@ -38,23 +38,26 @@
   },
   "homepage": "https://github.com/onchain-id/solidity#readme",
   "devDependencies": {
+    "@openzeppelin/contracts": "^3.2.0",
     "ansi-colors": "^4.1.1",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
+    "chai-bignumber": "^3.0.0",
     "chai-bn": "^0.2.0",
-    "ethereumjs-abi": "^0.6.8",
-    "ethereumjs-util": "^6.2.0",
-    "solidity-coverage": "^0.7.5",
-    "semver": "^6.3.0",
-    "solhint": "^2.3.0",
-    "solidity-docgen": "^0.3.14",
-    "truffle": "^5.1.13",
     "eslint": "^7.2.0",
     "eslint-config-airbnb-base": "^14.1.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-prettier": "^3.1.3",
+    "ethereumjs-abi": "^0.6.8",
+    "ethereumjs-util": "^6.2.0",
+    "ganache-cli": "^6.10.1",
     "prettier": "^2.0.5",
-    "ganache-cli": "^6.10.1"
+    "semver": "^6.3.0",
+    "solhint": "^2.3.0",
+    "solidity-coverage": "^0.7.5",
+    "solidity-docgen": "^0.3.14",
+    "truffle": "^5.1.13",
+    "truffle-proxy": "git+https://github.com/mdcoon/truffle-proxy.git"
```

Comment:
Why a git reference? This is very bad and a severe security issue.

Reply:
Removed, I am not using this package anyway.